storage/tsdb/tsdb.go

```diff
@@ -195,6 +208,27 @@ func Open(path string, l log.Logger, r prometheus.Registerer, opts *Options) (*t
 	return db, nil
 }
 
+// ChooseRetention is some roundabout code to support both RetentionDuration and Retention (for different flags).
+// If Retention is 15d, then it means that the default value is set and the value of RetentionDuration is used.
+func ChooseRetention(oldFlagDuration, newFlagDuration model.Duration) model.Duration {
+	defaultDuration, err := model.ParseDuration("15d")
```

Comment:
I see `"15d"` being used in 4 places now (1 more in test). Should be have a single variable for this?